web/src/app/pages/create-survey/loi-permissions/loi-permissions.component.ts

```diff
@@ -35,8 +43,12 @@ interface OptionCardConfig {
   templateUrl: './loi-permissions.component.html',
   styleUrls: ['./loi-permissions.component.scss'],
 })
-export class LoiPermissionsComponent {
-  readonly loiPermissionsControlKey = 'loiPermissionsMethod';
+export class LoiPermissionsComponent implements OnInit, OnChanges {
+  @Input() permissionsOption!: LoiPermissionsOption;
```

Comment:
Nit: Does this refer to the original select option when the component is initialized? If so maybe we can call it something that makes that obvious, e.g. `originalSelection`.